Elevel/Application/Features/QuestionCommands/UpdateQuestionCommand.cs

```diff
@@ -49,11 +54,22 @@ public Handler(IApplicationDbContext context, IMapper mapper)
             public async Task<Response> Handle(Request request, CancellationToken cancelationtoken)
             {
                 var question = await _context.Questions.FirstOrDefaultAsync(a => a.Id == request.Id, cancelationtoken);
+
                 if (question is null)
                 {
                     throw new NotFoundException($"Question with id {request.Id}");
                 }
 
+                var answerIds = question.Answers.Select(x => x.Id);
+
+                foreach (var answer in request.Answers)
+                {
+                    if (!answerIds.Contains((Guid)answer.Id))
+                    {
+                        throw new ValidationException($"This question doesn't contain an answer with id {answer.Id}");
+                    }
+                }
```

Comment:
It's correct but I would use Union

Reply:
if (request.Answers.Any(x => !answerIds.Contains(x.Id))